Remotion/Web/ClientScript/Scripts/SmartNavigation.ts

```diff
@@ -153,46 +153,45 @@ class SmartFocus
     var activeElementID = data;
     if (! StringUtility.IsNullOrEmpty (activeElementID))
     {
-      var activeElement = $('#' + activeElementID);
+      var activeElement = document.getElementById(activeElementID);
       if (activeElement)
       {
-        var isEnabledFilter = function (this: HTMLElement)
+        function isEnabledFilter(el: HTMLElement)
         {
-          var _this = $ (this);
-          if (_this.is ('button'))
+          if (el.tagName === 'BUTTON')
             return true;
-          if (_this.is ('input[type=button]'))
+          if (el.tagName === 'INPUT' && (el as HTMLInputElement).type.toLowerCase() === "button")
```

Comment:
In line 169, it's `el.matches(...)`, here it's separate checks. Why not use `el.matches(...)` at the other spots, too?

Reply:
The performance part of my brain would not let me. I changed the if statements to avoid overusing `.matches` and used it when there is no simple way converting the css selector to js or if the if-statement would get overcrowded.

Reply:
Ah, I.C. Performance is always helpful. I'll wait for the updated version when the other stuff's fixed, but maybe we should just resolve the other two `matches` to lower level lookups, too, since it's better performance and keeps things consistent within the IF-statement. Without symmetry, you always wonder if there's a special reason to it, beyond 'here it read better this way'.